tests/api/proquest/test_importer.py

```diff
@@ -1429,3 +1430,28 @@ def test_monitor_does_not_clean_removed_items_if_downloading_has_not_finished(se
 
         # Assert
         monitor._clean_removed_items.assert_not_called()
+
+    def test_that_errors_extracting_metadata_are_handled_successfully(self):
+
+        error_msg: str = "*test error*"
```

Comment:
Just an FYI: You don't need to add a type when that type will be inferred from the assignment.